test/runtests.jl

```diff
@@ -1007,6 +1007,12 @@ end
                         ("'infinity'::timestamptz", ZonedDateTime(typemax(DateTime), tz"UTC")),
                         ("'-infinity'::timestamptz", ZonedDateTime(typemin(DateTime), tz"UTC")),
                         # ("'epoch'::timestamptz", ZonedDateTime(1970, 1, 1, 0, 0, 0, tz"UTC")),
+                        ("DATE '2017-01-31'", Date(2017, 1, 31)),
+                        ("'infinity'::date", typemax(Date)),
+                        ("'-infinity'::date", typemin(Date)),
+                        ("TIME '13:13:13.131'", Time(13, 13, 13, 131)),
+                        ("TIME '01:01:01'", Time(1, 1, 1)),
+                        ("'allballs'::time", Time(0, 0, 0)),
```

Comment:
I had never heard this before and had to go look it up.

Reply:
It's one of my favourite silly names